### DISCUSSION TITLE
質問の他の人の回答にコメント出来るようにしてほしい

### DISCUSSION THREAD

**Discussion by radian-jp** [increments/qiita-discussions#362] [要望・アイデア・不具合]
明らかに間違った回答を訂正・補足したい場合などに、直接コメント出来ないのが不便に感じます。
例として、

https://qiita.com/wifrstfasnriov/questions/84898783c41b7b4be25d

質問者である @wifrstfasnriov さんが、間違った理解をして回答を書き込んでいますが、このような場合、直接その回答に対してコメントをつけたいです。機能として存在しないので、仕方なく自分の回答のコメントに追記してますが、その後第三者が見たときに回答に訂正のコメントが付いたかどうか気付くのは難しく、間違ったままの回答を参照する可能性もあり、時系列も判り難くなってしまいます。

**Comment by tanaka-asami415** [comment]
@radian-jp 
ありがとうございます🙏運営としても必要性を感じております。
質問機能のアップデートを予定しているため、その際に合わせて検討いたします！🙇

**Comment by blue32a** [comment]
私は現状の制限された状態にも良い点があると思っています。

1. やり取りがシンプルで、質問者が置いてきぼりになるような状態になりにくい
2. 回答に対する度を超えた訂正、批判が防止される（気がする）

とはいえ訂正に限らず第三者の立場からアクションを起こしたいときはあるので、安心して質問/回答できるような形で実現されると嬉しいです。

**Reply by yoshinoyuya** [reply]
@blue32a さん
目からウロコでした。
改めて考えた際、どんなことでも出来るシステムは、要望を叶え問題ないように見えます。
しかし、適度な制限や、不便さがあることで、使いやすくなることもありそうですね。

@tanaka-asami415 さん
解決の案としては、

1. 回答コメントに対して、「良い回答/悪い回答」good badのような評価などはどうでしょうか(Stack Overflowがそれに当たるかと思います。)
2. こちらは、微妙かもしれませんが、回答コメントに対して、編集リクエストを送るように出来るとかなど

どちらも、回答コメントに他者からアクションは起こせるけど、やりすぎない程度だと思います。

**Reply by radian-jp** [reply]
@yoshinoyuya さん

> 回答コメントに対して、「良い回答/悪い回答」good badのような評価などはどうでしょうか(Stack Overflowがそれに当たるかと思います。)

現状いいね機能はありますが、低評価も確かに出来た方がいいと思いますが、それはそれとして、やはり他者の回答でもコメントは付けれるべきです。低評価だけでコメントを付けれないのであれば、低評価の理由が判らなくて逆にモヤモヤすると思います。（むしろ低評価機能つけるなら、コメントか理由選択必須でもいい気がする）
ちなみに、Stack Overflowも他者の回答にコメントは付けれます。（むしろ付けれないサイトを見た事がない）

> 回答コメントに対して、編集リクエストを送るように出来るとかなど

編集リクエストは無視されてしまえば終わりなので、やはり全ての人に見える形で残すのが重要だと思います。

**Comment by tanaka-asami415** [comment]
ご意見ありがとうございます🙏

[開発ロードマップ](https://github.com/orgs/increments/projects/8/views/1)にも掲載しているのですが、既に
「質問者・回答者以外のユーザーも回答に対してコメントができるようにする」ことに着手しております！

また、安心して質問や回答できる場所が維持できるように、
回答に対するコメントのスパム検知や通報機能のリリースも予定しております。

ロードマップの説明についてはこちらをご参照ください https://github.com/increments/qiita-discussions/discussions/449

**Reply by blue32a** [reply]
@tanaka-asami415 
ありがとうございます。
進行中であることに気が付かずにコメントしてしまったのですが、このdiscussionはクローズ扱いなのでしょうか？
（discussionに慣れていないのでクローズがあるのか分かりませんか・・・）

**Reply by tanaka-asami415** [reply]
@blue32a 
途中経過が分かりづらく申し訳ありません🙏
着手中ですが、まだリリースをしていないためクローズしておりませんでした。

着手中であっても様々なご意見は常にお聞きしたいので、
コメントはお気軽にお寄せいただけますと大変ありがたいです！

これまではDiscussionにクローズの機能がなかったため、クローズラベルをつけて運用していたのですが、
最近クローズ機能が追加されたので、今後の運用方法は改めて検討いたします！🙇

**Comment by tanaka-asami415** [comment]
@radian-jp さん、みなさま
大変お待たせしました！🙇
質問者・回答者以外も回答に対してコメントできるようになりました✨
また、回答コメントを通報出来るようにもいたしました！

[リリースノート](https://qiita.com/release-notes/2023-03-24)

**Reply by radian-jp** [reply]
対応ありがとうございます！

**Reply by blue32a** [reply]
@tanaka-asami415 
ちなみに「質問のコメント」はContributionにどのような影響があるでしょうか？
補足や訂正ができるようになったとすれば、それらは貢献として考えられると思います。
https://help.qiita.com/ja/articles/qiita-contribution

**Reply by tanaka-asami415** [reply]
@blue32a 
おっしゃる通りです。質問のコメントにも「いいね」をできるようにしたいと考えているため
「いいね」をできるようにすることと合わせて現在検討しております！
今回のリリースのタイミングでできておらず申し訳ないです🙏

**Comment by tanaka-asami415** [comment]
本Discussionのリクエスト「質問の他の人の回答にコメント出来るようにしてほしい」については
リリースが完了したため、一旦こちらのDiscussionはクローズさせていただきます！
追加でリクエスト等ありましたら、別途Discussionを作成ください！🙏
